### PR TITLE
Created students controller,  index/show page, and routes

### PR DESCRIPTION
**Pull request by Jcvang5**
### Changes
Give a short summary of the changes made in this PR. 
Created students controller,  index/show page, and updated routing.

### Link to Issue 
https://app.asana.com/0/1206510424537042/1206645498622410
https://app.asana.com/0/1206510424537042/1206645498622412

### Comments, Notes, or  Questions
This section is if you want to share something related to the PR with the reviewer. 

### Related Screenshots 
For FE tickets specifcially, please include a screenshot of your work!

### Before you submit for review, Did you...

- [x] Added relevant tests (if applicable)
- [x] Linked Asana ticket to PR
- [x] Added comments, notes, or questions for reviewwer
- [x] Added relevant screenshots
- [x] Ensured ticket title is clear and concise(i.e. 'Donation Form Feature' instead of 'sfg/chels/donation-form'

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  get 'students', to: 'students#index'
+  get 'students/:id', to: 'students#show', as: 'student'
```
Comment (edithemmings) [inline config/routes.rb:5]: These can be simplified into 1 line:
```
  resources :usages, only: %i[index, show]
```

File: app/controllers/students_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+class StudentsController < ApplicationController
+  def index
+    @students = Student.all
```
Comment (edithemmings) [inline app/controllers/students_controller.rb:5]: We will eventually want this scoped to a single educator, but that can be done as an update for the person who works on the index

**Review by edithemmings — COMMENTED** [review]
Looks great! And such a quick turnaround -- especially for pair programming. I left one note as a reference, and one suggestion for simplifying the routes

File: app/controllers/students_controller.rb (hunk 2 above)
Comment (camckin10) [reply app/controllers/students_controller.rb:5]: @Jcvang5  you could add a commented out  TODO message such as, "TO DO: Scope to single educator."
Code comments are always helpful.

**Review by camckin10 — APPROVED** [review]
Looks good! 
I left a comment to add to your code. 
Please make sure you resolve all the conversations in this PR.

**Review by edithemmings — APPROVED** [review]
Looks great! Nice work @Jcvang5 and @asmanoor17!! 🎉

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 Rails.application.routes.draw do
+  resources :usages, only: %i[index show]
```
Comment (edithemmings) [inline config/routes.rb:4]: Whoops! I missed this originally, but this should have been `students`, not `usages`

**Pull request merged**